src/inverseRobot.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
+const robert = {
+  Robert: 'name',
+  123: 'chipVer',
+  113: 'chipVerPro',
+};
+
+inverseRobot(robert);
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (robot.hasOwnProperty(value)) {
+      inversedRobot[value] = key;
+    }
+
+    if (inversedRobot.hasOwnProperty(value)) {
+      return null;
+    }
+
+    inversedRobot[value] = key;
```

Comment:
```suggestion
    if (inversedRobot.hasOwnProperty(value)) {
      return null;
    }

    inversedRobot[value] = key;
```
Do you really need a first check?